infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesPods.java

```diff
@@ -229,7 +231,7 @@ public void onClose(KubernetesClientException cause) {
    *     </ul>
    *     otherwise, it must be explicitly closed
    */
-  public CompletableFuture<Void> waitAsync(String name, Predicate<Pod> predicate) {
+  public CompletableFuture<Void> waitAsync(String name) {
```

Comment:
Previously this method was abstract and it waits until predicate will be true. But now method is still named `wait` but wait what? Do you think it makes sense to rename it to something like `waitRunning`?

P.S. Please fix java doc and add information about succeeded and failed statuses